scmtfvc.properties

```diff
@@ -2,12 +2,18 @@ category=Integration
 description=Provides SCM TFVC integration
 devVersion=
 privateVersions=
-publicVersions=2.2
+publicVersions=2.3,2.2
 archivedVersions=1.0,2.0,2.1,2.1.1,2.1.2
 
 defaults.mavenGroupId=org.sonarsource.sonar-plugins.tfvc
 defaults.mavenArtifactId=sonar-scm-tfvc-plugin
 
+2.3.description=embed TFS API
+2.3.sqVersions=[7.3,LATEST]
```

Comment:
SQ versions <7.9 have been archived. Please start this range at 7.9.

Reply:
Mmh, I think it would work also with older versions.
Why do you want to stop it?
Anyway, I've changed it.